Makefile

```diff
@@ -45,7 +45,6 @@ ensure-important-modules-up-to-date:
 generate:
 	go generate ./pkg/... ./cmd/...
 	cd ./e2e && go generate ./...
-	$(MAKE) fmt
```

Comment:
I remember the reason we run fmt after go generate is fmt will reorder the imports in a different way with go generate.

Reply:
Yes, it is because the fixed file is not ignored by `make fmt`. Now we have fixed the file, `make fmt` will no longer try to format generated files, so we do not need to run `make fmt`